Resources/Prototypes/Catalog/VendingMachines/Inventories/theater.yml

```diff
@@ -5,6 +5,7 @@
   startingInventory:
     ClothingOuterWinterClown: 1
     ClothingOuterWinterMime: 1
+    ClothingMaskSexyClown: 1
```

Comment:
You forgot the jumpskirt too

Reply:
jumpskirt spawns when you select jumpskirt option